docs/users-git-versions.html

```diff
@@ -5,7 +5,38 @@
 ---
 
 <div class="chart-placeholder">
-	<h3>Git Versions</h3>
+	<h3>Recommended Git Versions</h3>
+	<canvas data-type="recommended-git-versions"></canvas>
+	<div class="info-box">
+		<p>
+			Shows how many users connected with recommended, outdated, and vulnerable Git clients.
```

Comment:
I am not sure ATM... does it work for SSH? It sure does for HTTPS.

In addition, maybe we should mention that the number are only correct for command line Git clients and UI's that use the command line Git clients under the hood (e.g. SourceTree). GitHub Desktop, SmartGit, etc might have a different name in the logs.